celo-org/wallet
# Add analytics event for feature flags

### Description

Just adding an event so we can tell which flags are on or off for users. This will help us in running AB tests and other analytics we might want to perform.

Part of: https://app.zenhub.com/workspaces/wallet-backlog-board-60092d006a84e3001d0d5936/issues/celo-org/wallet/697

### Other changes

None

### Tested

Not yet but working on getting a segment dev key. The steps are as follows:
1. Enable analytics on local dev in ValoraAnalytics.isEnabled
2. Set a test segment key in your env or directly in config.ts
3. Edit feature flags in the firebase console https://console.firebase.google.com/
4. Restart the app
5. Check segment logs or big query (note that it might take a while for the event to propagate, still trying to figure out how to make this more real time)

### How others should test

I think this is pretty difficult for the qa team to perform so not needed at the moment. Will chat with @MuckT to see what we can do here.

### Related issues

- Fixes #636 

### Backwards compatibility

Yes.
